### ISSUE TITLE
Redirect packages after renames

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/Coherence#93]
In absence of a first-class way to "rename" a package on nuget (like https://github.com/NuGet/Home/issues/1590), we should find a way to help users with package renames. It's hard to follow package renames.

Example:
```
beta8  EntityFramework.SqlServer 
rc1    EntityFramework.MicrosoftSqlServer 
rc2    Microsoft.EntityFrameworkCore.SqlServer
```

One possible approach: issue an update to an old package that somehow throws a build error that alerts user of the new package name.

Two most recent examples of people confused by our package renames:
https://github.com/aspnet/EntityFramework/issues/5281
https://github.com/aspnet/EntityFramework/issues/5324

**Comment by Eilon** [comment]
Very good idea but not planning to do this in bulk. If we see specific packages where people use the wrong name, we can reconsider this and either create redirect packages for those specific cases, or perhaps do the whole thing.
